driver/sources/src/main/scala/org/virtuslab/ideprobe/ProbeDriver.scala

```diff
@@ -222,6 +228,7 @@ class ProbeDriver(
     runTestsFromGenerated(runConfiguration, runnerToSelect = Some(runnerToSelect))
   }
 
+  // Is this method part of the API or should it be private???
   def runTestsFromGenerated(runConfiguration: TestScope, runnerToSelect: Option[String]): TestsRunResult = {
```

Comment:
It is part of the api. Useful for `runTestsFromGenerated(iJ.config[TestScope]("scope"), iJ.config.get[String]("runner"))`

Reply:
Right, in case of `awaitIdle` we make the method private but it there this kind api appears to be useful if the runner name is not provided in config.

Reply:
Okay, but IMO these three methods are redundant. I think the API should have either a method with a non-optional second argument and another method with just one argument, or just one method with an optional second argument, or maybe just a two-argument method with some default token value for the second argument.

Reply:
We cover 2 use cases: generic, and simple calls. We should rather have an api that can be used conveniently by the users.